Dev Portal: Try it CORS configuration (#2044)
Kong/developer.konghq.com: app/dev-portal/apis.md

If [RBAC](/dev-portal/security-settings/) is enabled, approved developers must be assigned to a [Team](/dev-portal/access-and-approval/) to access the API.

## Allow developers to try requests from the Dev Portal spec renderer

When you upload a spec for your API to Dev Portal, you can enable the **Try it!** feature that will allow developers to try your API right from Dev Portal. **Try it!** enables developers to add their authentication credentials, path parameters, and request body from the spec renderer in Dev Portal and send the request with their configuration. 

> To enable the **Try it!** feature, navigate to **ADD PORTAL NAVIGATION PATH HERE** in the {{site.konnect_short_name}} UI. In addition, you also need to enable the CORS plugin for this feature to function. See the following table to help you determine which CORS configuration to use depending on the Route associated with your APIs:

Author: ```suggestion
The **Try it!** feature is enabled by default. You also need to enable the CORS plugin for this feature to function. See the following table to help you determine which CORS configuration to use depending on the Route associated with your APIs:
```
